src/azure-cli/setup.py

```diff
@@ -90,7 +90,7 @@
     'azure-mgmt-eventhub~=2.6',
     'azure-mgmt-hdinsight~=1.3.0',
     'azure-mgmt-imagebuilder~=0.2.1',
-    'azure-mgmt-iotcentral~=1.0',
+    'azure-mgmt-iotcentral~=2.0.0',
```

Comment:
Is this the correct version? The publishing date looks correct but the features section doesn't seem to have your changes.
![image](https://user-images.githubusercontent.com/35089121/72093691-c1d50580-32c9-11ea-97e9-55e1579bca7d.png)

Reply:
Here is the link to the python SDK. https://pypi.org/project/azure-mgmt-iotcentral/

Reply:
build fail is also related to this
pip check
azure-cli 2.0.79 has requirement azure-mgmt-iotcentral~=2.0.0, but you have azure-mgmt-iotcentral 1.0.0.

Reply:
@r-delgadillo This is indeed the correct version, as I am able to create/update to ST1/ST2 in az dev environment with the version 2.0.0
@kawayiboy tu, already made new commits to fix it.